sdk/appcenter/src/test/java/com/microsoft/appcenter/http/HttpClientRetryerTest.java

```diff
@@ -153,6 +170,42 @@ public ServiceCall answer(InvocationOnMock invocationOnMock) {
         verifyNoMoreInteractions(call);
     }
 
+    @Test
+    public void delayUsingRetryHeader(){
+        long retryAfterMS = 1234;
```

Comment:
and comment setup, /* If */, /*When */ , /* Call */

Reply:
To add onto what @YiMysty said - generally:
```
/* If */
..setup code...
/* When */
... call the code/methods under test....
/* Then */
...do verification....
```